trustyai.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: trustyai
+title: TrustyAI
+permalink: /trustyai/
+---
+
+# TrustyAI Initiative
+<br/>
+
+The TrustyAI initiative aims to offer value-added services to the Kogito ecosystem to make it a trustworthy Business Automation solution. The project currently focuses on the open standard DMN and introspects the decision-making process using the following aspects:
+
+- **Explainability**: XAI algorithms to enrich model execution information
+- **Tracing and Accountability**: extract, collect and publish metadata for auditing and compliance
+- **Runtime Monitoring**: a service should be easily observable using dashboards both from a business and an operational perspective
+
+## Value-added services
+
+### Explanability
+TrustyAI aims to help explain black-box machine learning models by using XAI techniques. 
+Black-box models are not transparent about their processing, and the user only knows the inputs and outcomes of the system. 
+An example of this is a neural network; these models are more accurate for predictions; however, their complexity can make them hard to interpret. 
+
+XAI techniques are used within TrustyAI to introspect these black-box models to describe predictions and outcomes. 
+This is done by looking at a feature importance chart. This is where the model’s inputs are ordered by the most important ones for the decision-making process. 
+This can help to capture if a model is biased. Model biased can damage a company’s reputation. 
+For example, if you have a hiring machine learning system and the most important feature they looked at was gender, this would show a bias. 
+TrustyAI reduces the chance of this by showing the important features used in the machine learning model can reduce business risk.
+
+### Tracing and Accountability
+The Audit UI is a dashboard that targets business users/auditors, where each transaction is stored and can be analysed. 
+For each transaction, it is possible to access the user inputs, the outcomes produced and the explanation of each of them. 
+
+The Audit UI traces every decision made and helps to create accountability of the system. 
+Accountability is guaranteed by the caseworker who can access the outcomes from the decision. 
+This means they can ensure that the outcomes of the system meet the requirements of the company.
+
+### Runtime Monitoring
+Runtime monitoring allows for both business and operational metrics to be displayed in a Grafana dashboard configurable by the user. 
+Business monitoring dashboards are generated based on model information (DMN or machine learning models) so that users can monitor business aspects and have an aggregated view of decision behaviours. 
```

Comment:
```suggestion
Business monitoring dashboards are generated based on model information (DMN or machine learning models) so that users can monitor business aspects and have an aggregated view of decision behaviors. 
```

Reply:
British English :) Which is fine if that's how the website already is. Just need to be consistent. I don't think the consensus across KIE community is British though.